Makefile.am

```diff
@@ -17,3 +17,6 @@ EXTRA_DIST = \
 	RELEASENOTES.md
 
 AM_DISTCHECK_CONFIGURE_FLAGS=$(NCCL_NET_OFI_DISTCHCK_CONFIGURE_FLAGS)
+
+TAR_OPTIONS = --owner=0 --group=0
```

Comment:
This is what the automake manual recommends too, fwiw.
https://www.gnu.org/software/automake/manual/html_node/Basics-of-Distribution.html

Reply:
Added that to the comment